bootenv.go

```diff
@@ -20,13 +20,14 @@ import (
 	"strings"
 
 	"github.com/mendersoftware/log"
+	"github.com/mendersoftware/mender/cmd"
 )
 
 type uBootEnv struct {
-	Commander
+	cmd.Commander
 }
 
-func NewEnvironment(cmd Commander) *uBootEnv {
+func NewEnvironment(cmd cmd.Commander) *uBootEnv {
```

Comment:
Right. We are having some issues with naming various things. Perhaps something to take a look at before release.